dock/api.py

```diff
@@ -37,6 +38,7 @@ def build_image_in_privileged_container(build_image, git_url, image,
     :param push_buildroot_to: str, repository where buildroot should be pushed
     :param parent_registry_insecure: bool, allow connecting to parent registry over plain http
     :param target_registries_insecure: bool, allow connecting to target registries over plain http
+    :param pull_base_image: bool, download base image updates from registry
```

Comment:
could we change this? I would vote for something like `pulls or updates base image specified in dockerfile`